src/Boxer/Graphics/Shader.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "../Common.h"
+#include "../GL.h"

```

Comment:
Don't expose GL types here. There is no reason to do so. Use U64 instead of GLenum, etc. This prevents us from having to load the entire GL header every time this is included.

Plus, this will be easier to abstract/refactor later if we decide to implement Vulkan/DX.